### ISSUE TITLE
Question: why isn't `protos.json` loaded with a `require` call in Node environment?

### ISSUE THREAD

**Issue by hmaurer** [googleapis/nodejs-firestore#1493]
Hello!

During an attempt to bundle (via [esbuild](https://esbuild.github.io/)) a Node application using the `firebase-admin` SDK I noticed that `protos.json` is loaded using a `require` call in a browser environment, but loaded differently in a Node environment, c.f. https://github.com/googleapis/nodejs-firestore/blob/c7d67fb749aaf76050c08d29b4c6fca28ec9f5ce/dev/src/v1/firestore_admin_client.ts#L164

While I acknowledge that bundling Node applications may not be a use-case you want to support, I was wondering why you made the explicit choice of not using `require` in a Node environment? My suggestion would be to use `require` in all environments, like so:

```
this._protos = this._gaxGrpc.loadProto(require('../../protos/protos.json'));
```

It'd appear that I am not the only one who ran into this issue:
- https://github.com/netlify/next-on-netlify/issues/68
- https://github.com/serverless-nextjs/serverless-next.js/issues/876

Bundling back-end applications has some benefits (in my case, generate the smallest possible package for services deployed to Cloud Run / Cloud Functions), so if making this small change would enable it it'd be wonderful!

Thanks for your work.

-------

Related threads:
- https://github.com/googleapis/nodejs-speech/issues/489
- https://github.com/googleapis/google-cloud-node/issues/2933

**Comment by hmaurer** [comment]
Update: it looks like my suggestion has been implemented a few days ago: https://github.com/googleapis/gapic-generator-typescript/pull/861! There's only one question left then: when will this come to this package?

**Comment by alexander-fenster** [comment]
@hmaurer It came in today :)  #1500 is where it happened.

**Comment by hmaurer** [comment]
@alexander-fenster Wonderful, thanks! <3

**Comment by alexander-fenster** [comment]
Fixed by #1500.
